web/views.py

```
@@ -4755,71 +4769,540 @@ def delete_team_goal(request, goal_id):
+    # Check if user is teacher or enrolled student
+    is_teacher = request.user == classroom.teacher
+    is_enrolled = False
+    if classroom.course:
+        is_enrolled = classroom.course.enrollments.filter(student=request.user, status="approved").exists()
+
+    if not (is_teacher or is_enrolled):
+        messages.error(request, "You do not have access to this virtual classroom.")
+        return redirect("course_detail", slug=classroom.course.slug if classroom.course else "course_search")
+
+    # Get or create customization settings to prevent DoesNotExist errors
+    customization, created = VirtualClassroomCustomization.objects.get_or_create(
```

Ruff (0.14.0)

Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)